doc/signify.txt

```diff
@@ -325,16 +325,11 @@ Enable line highlighting in addition to using signs by default.
 The sign to use if a line was added, deleted or changed or a combination of
 these.
 
-NOTE: You can use unicode characters.
+You can use unicode characters, but signs mustn't take up more than two cells.
```

Comment:
I'd change `mustn't` to `must not`, since I generally try to avoid contractions for documentation.  Similarly for `you'll`.